spec/models/pet_contract_spec.rb

```diff
@@ -5,28 +5,17 @@
     subject(:find_by_contract_number) { described_class.find_by_contract_number(contract_number) }
 
     before do
-      client = described_class.send(:kb_client)
-      connection = client.send('connection')
-      connection.builder.adapter :test, stubs
+      stub_request(:get, "https://test_api_barkkb.com/v1/petcontracts/contractnumber/#{contract_number}")
+        .to_return(status: 200, body: contract_resource.to_json)
```

Comment:
I had to change this test since [the build failed](https://app.codeship.com/projects/e264d409-17ad-4a39-ad60-4e2a6c75344d/builds/19e48ccf-7bae-4e13-a632-91c7b0f78b8b?component=2cf5b442-b044-4a30-837e-9a57a17279fb_1694790521_combined) when the other tests were added.
I don't really understand why.